lxd/main_sql.go

```diff
@@ -161,7 +161,7 @@ func sqlPrintSelectResult(format string, result internalSQLResult) error {
 	for _, row := range result.Rows {
 		r := make([]string, 0, len(row))
 		for _, col := range row {
-			r = append(r, fmt.Sprintf("%v", col))
+			r = append(r, fmt.Sprint(col))
```

Comment:
i got hit by this bug today i think, so good spot.

Reply:
This is a bug? Also credit to @simondeziel for spotting this one!

Reply:
yeah i think so, or something similar as was  using `%` wildcards in my query and then i was seeing the unmatched parameter message.